hypothesis/client
# Implement moderation UI for group reader for flagging an annotation.

To test:
Click on the flag icon in an annotation card. The icon will change colour and the title will change too.

Note:
Displaying flag status of annotation will be implemented as part of a different PR.
Also, This is not behind a feature flag. Instead, we are limiting this feature to only third party users for now. See: https://hypothes-is.slack.com/archives/C4K6M7P5E/p1489766147808575

https://github.com/hypothesis/product-backlog/issues/181
